openlr_dereferencer/decoding/tools.py

```diff
@@ -61,3 +63,25 @@ def project(line_string: LineString, coord: Coordinates) -> float:
 def linestring_coords(line: LineString) -> List[Coordinates]:
     "Returns the edges of the line geometry as Coordinate list"
     return [Coordinates(*point) for point in line.coords]
+
+
+def compute_bearing(
```

Comment:
From debugging, this seems to return a value between `-180` and `+180`.  I think its better to normalise in the range `0` to `360`.

Reply:
Done now.